pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -735,6 +743,43 @@ type TiCDCConfig struct {
 	LogFile *string `toml:"log-file,omitempty" json:"logFile,omitempty"`
 }
 
+// TiProxySpec contains details of TiProxy members
+// +k8s:openapi-gen=true
+type TiProxySpec struct {
+	ComponentSpec               `json:",inline"`
+	corev1.ResourceRequirements `json:",inline"`
+
+	// Specify a Service Account for TiProxy
+	ServiceAccount string `json:"serviceAccount,omitempty"`
+
+	// The desired ready replicas
+	// +kubebuilder:validation:Minimum=0
+	Replicas int32 `json:"replicas"`
+
+	// TLSClientSecretName is the name of secret which stores tidb server client certificate
+	// which used by Dashboard.
```

Comment:
`which used by Dashboard.` is not accurate in `TiProxySpec`?

Reply:
Done